content/docs/administration/display.md

```diff
@@ -43,21 +47,20 @@ It is possible to use a custom logo for the site. This shows in the left navigat
 
 ![Custom logo login](/docs/administration/images/custom_logo_nav.png)
 
-The logo will need to be in the `svg` format. To update, simply 
+The logo will need to be in the `svg` format. To update, simply
 
-* enable the custom logo by clicking the toggle switch
-* copy the text of the .svg file and paste it into the text area
-* click save
+- enable the custom logo by clicking the toggle switch
```

Comment:
note: change isn't required. markdown supports both asterisk and dash as bullet point indicators, same as github

Reply:
Yeah sorry, that's just my markdown formatter having a field day 🤷‍♀️